beta.py

```diff
@@ -1,22 +1,7 @@
 #!/usr/bin/env python3
-import argparse
+import argparse, random, bcrypt, requests, os, sys, string
```

Comment:
Imports should be sorted into three groups:
1. import from standard library
2. import from third-parties
3. your own imports